java/javadoc/src/org/netbeans/modules/javadoc/search/IndexBuilder.java

```diff
@@ -417,6 +418,12 @@ private void handleTitleTag() throws IOException {
                             } else {
                                 title = new String(buf, 0, offset - 7, charset).trim();
                             }
+                            
+                            // Unescape the title, done for JDK 11.
+                            // JDK 11 title contains &amp;
+                            // NOI18N
```

Comment:
The NOI18N comment does not belong here, the comment is IMHO unnecessary, as "unescapeHTML" is pretty self-explaining.

Reply:
I am fixing it.

Reply:
Removed NOI18N comment.